### ISSUE TITLE
false positive on @pandacss/no-physical-properties

### ISSUE THREAD

**Issue by azu** [chakra-ui/eslint-plugin-panda#133]
When I write the following `cva` variants, `@pandacss/no-physical-properties` reports an error.

- https://github.com/chakra-ui/eslint-plugin-panda/blob/main/docs/rules/no-physical-properties.md

```ts
const heading = cva({
  base: {
    fontWeight: "strong",
    lineHeight: "m",
  },
  variants: {
    size: {
      "2xl": {
        fontSize: "2xl",
      },
      xl: {
        fontSize: "xl",
      },
      l: {
        fontSize: "l",
      },
      m: {
        fontSize: "m",
      },
      s: {
        fontSize: "s",
      },
      xs: {
        fontSize: "xs",
      },
    },
    align: {
      // false positive lint error is here
      left: {
        textAlign: "left",
        textWrap: "wrap",
      },
      center: {
        textAlign: "center",
        textWrap: "balance",
        wordBreak: "[auto-phrase]",
      },
      // false positive lint error is here
      right: {
        textAlign: "right",
        textWrap: "wrap",
      },
    },
    weight: {
      strong: {
        fontWeight: "strong",
      },
      regular: {
        fontWeight: "regular",
      },
    },
  },
  defaultVariants: {
    size: "2xl",
    align: "left",
    weight: "strong",
  },
});
```

Reported Error by `@pandacss/no-physical-properties`

```
./src/app/Heading.tsx
51:7  Error: Use logical property of `left` instead. Prefer `insetInlineStart`  @pandacss/no-physical-properties
60:7  Error: Use logical property of `right` instead. Prefer `insetInlineEnd`  @pandacss/no-physical-properties
```

## Reproduce Repository

- https://github.com/azu/panda-css-physical-properties-bug

```bash
git clone https://github.com/azu/panda-css-physical-properties-bug
cd panda-css-physical-properties-bug
pnpm install
pnpm run lint
./src/app/Heading.tsx
51:7  Error: Use logical property of `left` instead. Prefer `insetInlineStart`  @pandacss/no-physical-properties
60:7  Error: Use logical property of `right` instead. Prefer `insetInlineEnd`  @pandacss/no-physical-properties

```

## Actual

`@pandacss/no-physical-properties` report an error.

> 51:7  Error: Use logical property of `left` instead. Prefer `insetInlineStart`  @pandacss/no-physical-properties
> 60:7  Error: Use logical property of `right` instead. Prefer `insetInlineEnd`  @pandacss/no-physical-properties

## Expected

No Error

This `left` and `right` of cva's variants are not CSS physical property.
Perhaps it is a false positive because the names are the same.

**Comment by anubra266** [comment]
Just released a fix in [0.1.10](https://github.com/chakra-ui/eslint-plugin-panda/releases/tag/%40pandacss%2Feslint-plugin%400.1.10) @azu. Thanks for pointing it out
